src/coreclr/inc/utilcode.h

```diff
@@ -3743,8 +3615,10 @@ class MethodNamesList : public MethodNamesListBase
 
 #if !defined(NO_CLRCONFIG)
```

Comment:
Nit: `NO_CLRCONFIG` does not seem to be defined anywhere. This ifdef can be deleted.

Reply:
Will do.